### PR TITLE
fix-gray-failures-fast-using-automation-and-route53-arc-zonal-shift

### PR DESCRIPTION
**Pull request by CryptikCode**
# Content Pull Request Checklist

Please check to make sure your content submission meets the following requirements by adding an `x` between the brackets:

- [X] The title of this pull request reflects the title of my content.
- [X] I have reviewed the [Content Review Checklist](https://github.com/build-on-aws/content/blob/main/CONTENT_REVIEW_CHECKLIST.md).
- [X] This pull request contains one piece of content (post or page).

PR created  from https://github.com/build-on-aws/content/pull/372 

**By submitting this pull request, I confirm I own this content, have permission to use each image included in this content, and that it will be released under the [CC BY-SA 4.0 SA License](/LICENSE).**

### CODE REVIEW

File: posts/fix-gray-failures-fast-using-automation-and-route53-arc-zonal-shift/index.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+```bash
+cdk destroy
+```
+
+## Conclusion
+
+This blog post discusses gray failures, how to detect them, and how to reduce the end user impact by quickly responding to gray failures. A gray failure can affect the overall availability of an application. This could impact the end user experience and drive down the application traffic/usage resulting in loss of revenue. By using the concept of differential observability and availability zone independence architectures, you can quickly detect and respond to a zonal gray failure with reduced impact to the end user experience and also prevent possible loss of revenue. Automating the [Amazon Route 53 Application Recovery Controller Zonal Shift](https://docs.aws.amazon.com/r53recovery/latest/dg/arc-zonal-shift.html?sc_channel=el&sc_campaign=resiliencewave&sc_geo=mult&sc_country=mult&sc_outcome=acq&sc_content=arc-zonal-shift) feature helps decrease the mean-time-to-respond with little or no user intervention. This improves the overall resilience posture of the application and can help keep end users engaged. This blog post also provides a sample solution and a walk through on how to get started with automating zonal shift.
```
Comment (damonk10) [inline posts/fix-gray-failures-fast-using-automation-and-route53-arc-zonal-shift/index.md:176]: Can you add some links to other content that curious readers may be interested in to learn more?

File: posts/fix-gray-failures-fast-using-automation-and-route53-arc-zonal-shift/index.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+6. To verify the shift, log back into the Load Balancer page of the AWS Console and verify the status of the NLB. Note that new columns have appear for Zonal Shift as indicated below.
+
+   ![post-shift](images/after-zonal-shift.png)
+
+### Cleaning Up
+
+```bash
+cdk destroy
+```
+
```
Comment (damonk10) [inline posts/fix-gray-failures-fast-using-automation-and-route53-arc-zonal-shift/index.md:173]: Add a short paragraph here just describing on a high level what the reader accomplished with these steps. What's just happened?

File: posts/fix-gray-failures-fast-using-automation-and-route53-arc-zonal-shift/index.md (hunk 1 above)
Comment (CryptikCode) [reply posts/fix-gray-failures-fast-using-automation-and-route53-arc-zonal-shift/index.md:178]: Links added.

File: posts/fix-gray-failures-fast-using-automation-and-route53-arc-zonal-shift/index.md (hunk 2 above)
Comment (CryptikCode) [reply posts/fix-gray-failures-fast-using-automation-and-route53-arc-zonal-shift/index.md:173]: Paragraph added.

**Review by cobusbernard — APPROVED** [review]

**Pull request merged**